apikey-service/src/test/java/eu/europeana/apikey/controller/ApikeyControllerTest.java

```diff
@@ -49,12 +49,12 @@
     @Before
     public void setup()throws Exception {
         Apikey apikey = new Apikey(EXISTING_API_KEY, "edward",
-                "potts", "potts@mail.com");
+                "potts", "potts@mail.com","appNme", "company");
         apikey.setKeycloakId(EXISTING_API_KEY);
         apikeyRepo.saveAndFlush(apikey);
 
         apikey = new Apikey(DEPRECATED_API_KEY, "frank",
-                "sinatra", "sinatra@mail.com");
+                "sinatra", "sinatra@mail.com","appName", "company");
```

Comment:
I propose to also add a test case where the appname and company are null, because we have quite a bit of those in our current database